examples/io/open_save/README.rst

```diff
@@ -0,0 +1,2 @@
+Open and save from files
```

Comment:
In the doc, this appears after import/export. I would reverse that order, or even maybe move import/export as last, because I think it's less useful.

Reply:
Also in this subsection, the order is 1) open vector, 2) parse, 3) open raster.
I would have 1) raster 2) vector 3) parse.
I don't know if that's easy to change.

Reply:
Subsections now sorted explicitly, and examples by name instead of code length (default)